views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/uiElements.js

```diff
@@ -404,12 +404,56 @@ define([
         return mediaStimulus;
     }
 
+    /**
+     * Creates a countdown timer as a pie chart
+     * @param {Object} config
+     * @param {Object} config.$container - jQuery element that the countdown timer will be appended to
+     * @param {Number} config.delayInSeconds - delay in seconds
+     */
+    function countdownPieChartFactory(config) {
+        var countdownPieChart,
+            $container   = config.$container,
+            delay = config.delayInSeconds - 1,
+            $countdownPieChart = $(
```

Comment:
We usually declare vars one by line.
`var countdownPieChart:`
`var $container   = config.$container,`
`...`